README.md

```diff
@@ -71,3 +71,43 @@ A monitoring agent which runs on Linux machines and collects system metrics.
     `localhost:3000`
 
 * Login using _admin_ for both username and password.
+
+
+# Home
```

Comment:
Good work @BBBBliW. I only have two requested changes and a NIT:

### Requested Changes
* Could you put the sketches documentation inside the `docs` folder. I have created a `sketches.md` folder inside of docs where you can reference the links.
* @ife0ye @BBBBliW before closing this request, it could be useful if you guys come up with the sketch for the homepage with the _list of N machines_, that we discussed at Microsoft.

### NITs
* This is not essential and can be done later on, but if you wanted to further document your work, you could do something similar to what I did in my project last year in the frontend. Here is the link for reference: https://github.com/MaxCunningham19/legal_spellcheck/tree/main/resources/fe